docs/content/docs/concepts/users-accounts.mdx

```diff
@@ -380,6 +380,37 @@ export const auth = betterAuth({
 });
 ```
 
+#### Existing User Auto-Linking Policy
+
+The `existingUserMode` option controls when OAuth/SSO providers can auto-link to an existing user with the same email address. This provides fine-grained control over account linking security.
+
+```ts title="auth.ts"
+export const auth = betterAuth({
+    account: {
+        accountLinking: {
+            enabled: true,
+            existingUserMode: "trusted_providers_only", // recommended
+        }
+    },
+});
+```
+
+Available modes:
+
+- **`"never"`**: Never auto-link. Users must explicitly link accounts (e.g., via a signed-in "link account" flow).
+
+- **`"trusted_providers_only"`** (recommended): Only auto-link if the provider passes a trust check. This means the provider must be listed in `trustedProviders`. For SSO providers, domain verification also counts as a trust signal.
```

Comment:
I wonder if we could rephrase this in a way that is not specific to SSO and then we can further elaborate on the SSO plugin docs (which I think we already do).


```suggestion
- **`"trusted_providers_only"`** (recommended): Only auto-link if the provider passes a trust check. This means the provider must be listed in `trustedProviders` or the associated domain has been verified.
```